.github/workflows/ci.yml

```diff
@@ -200,7 +201,6 @@ jobs:
       fail-fast: false
     env:
       SOLVER: ${{ matrix.solver }}
-      OPAM_REPO_SHA: 02ead5a557
```

Comment:
Yes, it was to have a more up to date repo for 0install, not updated the general on because we needed it only here and not rebuild caches. lgtm!